conceptual/Introduction/WhatsNew/whats-new-20241.md

```diff
@@ -0,0 +1,30 @@
+---
+uid: whats-new-20241
+title: "What's New in PostSharp 2024.1"
+product: "postsharp"
+categories: "PostSharp;AOP;Metaprogramming"
+summary: "PostSharp 2024.1 introduces a consolidated Visual Studio extension for Metalama and PostSharp, along with support for CodeLens."
+---
+# What's New in PostSharp 2024.1
+
+PostSharp 2024.1 introduces a consolidated Visual Studio extension for Metalama and PostSharp, along with support for CodeLens.
+
+## Visual Studio Tools for Metalama and PostSharp
+
+* The Visual Studio extension for PostSharp has been consolidated with the extension for Metalama. Now renamed as [Visual Studio Tools for Metalama and PostSharp](https://marketplace.visualstudio.com/items?itemName=PostSharpTechnologies.PostSharp), this single extension can be used to work with both [Metalama](https://www.postsharp.net/metalama) and/or [PostSharp](https://www.postsharp.net/il).
+
+* CodeLens now displays details of how PostSharp enhances your code. This feature, previously available for Metalama in its former extension, has now been introduced to PostSharp. This feature replaces the underlining of enhanced code and related information in tooltips. If you miss this replaced feature, it can still be enabled in the Visual Studio Tools for Metalama and PostSharp options.
```

Comment:
```suggestion
* CodeLens now displays details of how PostSharp enhances your code. This feature, previously available for Metalama in its former extension, has now been introduced to PostSharp. CodeLens replaces the previous mechanism based on code underlining and tooltips. If you miss the old mechanism, it can still be enabled in the Visual Studio Tools for Metalama and PostSharp options.
```